src/Tribe/Query.php

```diff
@@ -342,7 +342,7 @@ public static function pre_get_posts( $query ) {
 
 				$skip_event_display_filters = is_admin() && $query->is_main_query() && ! tribe_is_ajax_view_request();
 
-				//@todo stop calling EOD cutoff transformations all over the place
+				//@todo [BTRIA-607]: Stop calling EOD cutoff transformations all over the place.
```

Comment:
:no_entry_sign: No space found before comment text; expected "// @todo [BTRIA-607]: Stop calling EOD cutoff transformations all over the place." but found "//@todo [BTRIA-607]: Stop calling EOD cutoff transformations all over the place."

<sub>hash: 7c8895804607610926f27d857eb35730</sub>